index.js

```diff
@@ -14,4 +14,165 @@ menuItem.forEach((menuItem) => {
+    technologies: ['HTML', 'CSS', 'JavaScript', 'Ruby', 'Bootstrap'],
+    technologiesTwo: ['HTML', 'Ruby on rails', 'CSS', 'JavaScript'],
+    desc: 'Exploring the future of media in Facebooks first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.',
+    liveLink: '',
+    sourceLink: '',
+  },
+  {
+    name: 'UBER Navigation',
+    description: 'Lorem Ipsum is simply dummy text of the prinhe 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent.',
+    image: './img/Dbg4.png',
+    stack: ['UBER', 'Full Stack Dev', '2018'],
+    technologies: ['HTML', 'CSS', 'JavaScript', 'Ruby ', 'Bootstrap'],
+    technologiesTwo: ['HTML', 'Ruby on rails', 'CSS', 'JavaScript'],
+    desc: 'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.',
+    liveLink: '',
+    sourceLink: '',
+  },
+];
+
+const cardsContainer = document.querySelector('#About');
+
+function getTechnologyClass(technology) {
+  switch (technology) {
+    case 'HTML':
+      return 'tech-t';
+    case 'CSS':
+      return 'tech-t';
+    case 'JavaScript':
+      return 'tech-j';
+    case 'Ruby on rails':
+      return 'tech-r';
+    default:
+      return 'tech';
+  }
+}
+
+projects.forEach((project) => {
+  const card = document.createElement('div');
+  card.classList.add('about-section');
+
+  card.innerHTML = `
+  <img src="${project.image}" class="card-img" />
+  <div class="canopy">
+    <h4 class="about-head">${project.name}</h4>
+    <ul class="cano">
+      ${project.stack.map((theme) => `<li class="tech">${theme}</li>`).join('')}
+    </ul>
+    <p class="canopy-txt">
+      ${project.desc}
+    </p>
+    <ul class="tech1">
+      ${project.technologiesTwo.map((technology) => `<li class="${getTechnologyClass(technology)}">${technology}</li>`).join('')}
+    </ul>
+    <button type="button" class="project-btn">See Project</button>
+  </div>
+`;
+
+  cardsContainer.appendChild(card);
+
+  const modalContainer = document.createElement('div');
+  modalContainer.classList.add('modal');
+  modalContainer.id = 'modal';
+
+  modalContainer.innerHTML = `
+  <div class="modal-content">
+    <span class="close">&times;</span>
+    <h2 class="project-title" id="projectTitle"></h2>
+    <ul id="cano">
+    </ul>
+    <div class="project-details">
+      <img class="project-image" id="projectImage" src="" alt="">
+      <div class="p-cont">
+      <p class="project-description" id="projectDescription"></p>
+      <div class="p-div">
+      <ul class="project-technologies" id="projectTechnologies"></ul>
+      <div class="project-buttons">
+      <button class="btn-link"><a class="project-live-link" id="projectLiveLink" href="#" target="_blank">See Live <img src="img/Icon.png" class="btn-img"/></a></button>
```

Comment:
- [ ] As suggested by the previous reviewers, It would be nice to remove the `button` tag and use the `a` tag to link to your live and source code with the `href` attribute.

- [ ] Kindly make sure your buttons look identical to the Figma design to follow the project requirement as suggested by the previous reviewers. Wright now the button icons are not well aligned with the text.
### Your design :point_down: 
![](https://user-images.githubusercontent.com/107192076/248309799-60459149-4ea9-4bd9-903b-c211f4888300.png)
### What it's supposed to look like :point_down: 
![](https://user-images.githubusercontent.com/107192076/248309778-78d60fb9-edf3-4743-9cdb-fa53380a052d.png)